composer/utils/checkpoint.py

```diff
@@ -566,6 +566,21 @@ def dist_cp_load(
                 planner=load_planner,
             )
         except CheckpointException as e:
+            if "torch.Size([816])" in str(e) and "rng" in str(e) and "cuda" in str(e):
```

Comment:
I'd match on a bit more of the string. For reference the full error is `Size mismatch between saved torch.Size([816]) and current: torch.Size([16]) for rng.0.cuda`. like regex * torch.Size([816]) * torch.Size([16]) * rng.\d.cuda (not a valid regex, just getting the point across)